docs/blocks/blocks.md

```diff
@@ -35,6 +35,41 @@ Blocks must be [registered][registering] to function.
 
     When a block is registered, *only* a block is registered. The block does not automatically have an `ItemBlock`. To create a basic `ItemBlock` for a block one should use `new ItemBlock(block).setRegistryName(block.getRegistryName())`. The unlocalized name is the same as the block's. Custom subclasses of `ItemBlock` may be used as well. Once an `ItemBlock` has been registered for a block, `Item.getItemFromBlock` can be used to retrieve it. `Item.getItemFromBlock` will return `null` if there is no ItemBlock for the Block, so if you are not certain that there is an ItemBlock for the Block you are using, check for null.
 
+Coloring a Block
+----------------
+
+For some blocks, Minecraft paints one texture with different colors. Color of grass, vines, waterlilys and what not changes depending on the biome it's placed in.
+
+### Block Color Handlers
+
+Block color handler is just an instance of `IBlockColor`. The interface itself contains only one method. Its full description follows.
```

Comment:
I would reword this as

"A block color handler is an instance of `IBlockColor`, which adds the `colorMultiplier` method below."

or something.